packages/cfd/src/Components/Routes/binary-link.jsx

```diff
@@ -1,32 +0,0 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import { NavLink } from 'react-router-dom';
-import { PlatformContext } from '@deriv/shared';
-import getRoutesConfig from '../../Constants/routes-config';
-import { findRouteByPath, normalizePath } from './helpers';
```

Comment:
@hamza-deriv I guess `findRouteByPath` and `normalizePath` are dead code now, Would you check if we can remove them too 🙏🏻

Reply:
Done